changes.go

```diff
@@ -780,8 +780,15 @@ func (s *ChangesService) change(tail string, changeID string, input interface{})
 
 	v := new(ChangeInfo)
 	resp, err := s.client.Do(req, v)
+	if err != nil {
+		return v, resp, err
```

Comment:
Other similar lines (above) do `return nil, resp, err`. Better to be consistent.

Reply:
Good one!